src/main/java/com/purbon/kafka/topology/model/Project.java

```diff
@@ -10,6 +10,47 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * Project model representation.
+ *
+ * YAML:
+ * <pre>
+ * {@code
```

Comment:
what is the benefit of adding this change here? As we change the model from time to time, I wonder, can we instead use a link to the doc folder? or generate this from the code?

Reply:
Good point. A link can work here as well. 
Just to have a way to have sample usages helps here, as there is some inheritance adding `principal` and `metadata`. 

If there are existing examples you have in mind to link, I can replace. Otherwise, I can create one.

Reply:
I think the files in the example-folder should be ones maintained for docs on topology (generally provided for users). The doc pages are only touching on the subject, and I think well maintained examples together with some docs are the best. Better have some links to files in example-folder from the doc pages.

For developers contributing to the code the tests and test-topology files helps for even more in dept understanding.

There will be many variations of topology for the different classes in the model so I don't think linking to the example files from Javadoc helps a lot.